internal/language/internal/rust/generate.go

```diff
@@ -47,7 +49,25 @@ func Generate(ctx context.Context, library *config.Library, sources *config.Sour
 	if err != nil {
 		return err
 	}
-	return sidekickrust.Generate(model, library.Output, sidekickConfig)
+	if err := sidekickrust.Generate(model, library.Output, sidekickConfig); err != nil {
+		return err
+	}
+	if err := command.Run("taplo", "fmt", filepath.Join(library.Output, "Cargo.toml")); err != nil {
+		return err
+	}
+	rsFiles, err := filepath.Glob(filepath.Join(library.Output, "src", "*.rs"))
+	if err != nil {
+		return err
+	}
+	if len(rsFiles) > 0 {
+		// rustfmt defaults to 2015 edition when run directly on files. Specify
+		// 2024 to match the edition in Cargo.toml.
+		args := append([]string{"--edition", "2024"}, rsFiles...)
+		if err := command.Run("rustfmt", args...); err != nil {
+			return err
+		}
+	}
```

Comment:
Do we know what the package name is? If we do we can run this:

```suggestion
    if err := command.Run("cargo", "fmt", "-p", ${thing that has the Rust package name}); err != nil {
        return err
    }
```